test/model/store_test.dart

```diff
@@ -157,18 +158,89 @@ void main() {
     await check(future).throws<AccountNotFoundException>();
   }));
 
+  test('GlobalStore.perAccount loading succeeds', () => awaitFakeAsync((async) async {
+    final globalStore = UpdateMachineTestGlobalStore(
+      globalSettings: eg.globalSettings(), accounts: [eg.selfAccount]);
+    final connection = globalStore.apiConnectionFromAccount(eg.selfAccount) as FakeApiConnection;
+
+    NotificationService.instance.token = ValueNotifier('asdf');
+    addTearDown(NotificationService.debugReset);
+
+    final future = globalStore.perAccount(eg.selfAccount.id);
+    check(connection.takeRequests()).length.equals(1); // register request
+    await future;
```

Comment:
nit: let's tighten the comparison with the tests below by rearranging slightly, like so:
```suggestion
    NotificationService.instance.token = ValueNotifier('asdf');
    addTearDown(NotificationService.debugReset);

    final globalStore = UpdateMachineTestGlobalStore(
      globalSettings: eg.globalSettings(), accounts: [eg.selfAccount]);
    final connection = globalStore.apiConnectionFromAccount(eg.selfAccount) as FakeApiConnection;
    final future = globalStore.perAccount(eg.selfAccount.id);
    check(connection.takeRequests()).length.equals(1); // register request

    await future;
```